src/specs/lineSpec.ts

```diff
@@ -153,18 +154,26 @@ export function genAnnotationLayer(
             field: 'end',
             type: 'temporal',
           },
+          color: {
+            condition: {
+              test: 'datum.uncertainty',
+              value: '#767676',
+            },
+            value: '#D46B08',
+          },
         },
       },
       {
         mark: {
           type: 'text',
-          color: '#D46B08',
           text:
             annotations.length > 1
               ? {
-                  expr: `'⚠ (' + (datum.index + 1) + ')'`,
+                  expr: `(datum.uncertainty ? '🛈' : '⚠') + ' (' + (datum.index + 1) + ')'`,
                 }
-              : '⚠',
+              : {
+                  expr: `datum.uncertainty ? '🛈' : '⚠'`,
```

Comment:
are the `🛈` characters an encoding problem?

Reply:
hm I see the proper icon: https://emojipedia.org/circled-information-source/

Reply:
Wacky. I'm on Mac OS 10.14.6 (which is a bit old) Chrome 95.0.4638.69 and I see the following:

![image](https://user-images.githubusercontent.com/1158666/144075969-911c987c-9ce6-4413-b627-115c7de776ec.png)

& it shows up in the OS's character viewer as blank:

![image](https://user-images.githubusercontent.com/1158666/144076156-3e65fb7a-5e46-4611-8966-e17d392cf417.png)

@ryantibs if the circled-i icon doesn't show up on some computers, is this display still acceptable?

Reply:
I switched to https://emojipedia.org/information/. since it just about the tiny icon on the top left of the gray area it doesn't matter that much